src/worker/browser-env/runner/test-runner/index.ts

```diff
@@ -176,7 +176,7 @@ export class TestRunner extends NodejsEnvTestRunner {
         });
 
         const timeout = this._config.urlHttpTimeout || this._config.httpTimeout;
-        const uri = new URI(this._config.baseUrl).query({ runUuid: this._runUuid }).toString();
+        const uri = urljoin(this._config.baseUrl, this._runUuid);
```

Comment:
It would be unclear, what that number means.

Perhaps, we should do something like this:
```ts
const uri = urljoin(this._config.baseUrl, `run-uuid-${this._runUuid}`);
```
So we get something like `http://foo.bar/run-uuid-12345`

Reply:
we agreed to use an URL like this `/run-uuids/:uuid` in the chat